### PR TITLE
[Api/Tizen] restricted elements in Tizen

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Update the restricted elements in Tizen.

Restricted elements
- capsfilter input-selector output-selector queue tee valve
- appsink appsrc
- audioconvert audiorate audioresample audiomixer
- videoconvert videocrop videorate videoscale videoflip videomixer compositor
- fakesrc fakesink filesrc filesink
- audiotestsrc videotestsrc
- jpegparse jpegenc jpegdec pngenc pngdec
- tcpclientsink tcpclientsrc tcpserversink tcpserversrc udpsink udpsrc
- xvimagesink ximagesink evasimagesink evaspixmapsink glimagesink (mmfw ini)
- theoraenc lame vorbisenc wavenc volume oggmux avimux matroskamux (mmfw ini)
- v4l2src avsysvideosrc camerasrc fimcconvert (mmfw ini)

Signed-off-by: Jaeyun Jung <jy1210.jung@samsung.com>

### CODE REVIEW

File: packaging/nnstreamer.spec [hunk 1]
```diff
@@ -179,6 +179,22 @@ Developmental kit for Tizen Native new single-shot NNStreamer API.
 %define api -Denable-capi=false
 %endif
 
+# Element restriction in Tizen
+%if %{with tizen}
+%define restricted_element	'capsfilter input-selector output-selector queue tee valve appsink appsrc audioconvert audiorate audioresample videoconvert videocrop videorate videoscale fakesrc fakesink filesrc filesink audiotestsrc videotestsrc jpegparse jpegenc jpegdec pngenc pngdec tcpclientsink tcpclientsrc tcpserversink tcpserversrc udpsink udpsrc xvimagesink ximagesink evasimagesink evaspixmapsink glimagesink theoraenc lame vorbisenc wavenc volume oggmux avimux matroskamux avsysvideosrc camerasrc fimcconvert'
+
```
Comment (myungjoo) [inline packaging/nnstreamer.spec]: add v4l2src, which is used instead of camerasrc by a few Tizen devices.

File: packaging/nnstreamer.spec [hunk 2]
```diff
@@ -179,6 +179,22 @@ Developmental kit for Tizen Native new single-shot NNStreamer API.
 %define api -Denable-capi=false
 %endif
 
+# Element restriction in Tizen
+%if %{with tizen}
+%define restricted_element	'capsfilter input-selector output-selector queue tee valve appsink appsrc audioconvert audiorate audioresample videoconvert videocrop videorate videoscale fakesrc fakesink filesrc filesink audiotestsrc videotestsrc jpegparse jpegenc jpegdec pngenc pngdec tcpclientsink tcpclientsrc tcpserversink tcpserversrc udpsink udpsrc xvimagesink ximagesink evasimagesink evaspixmapsink glimagesink theoraenc lame vorbisenc wavenc volume oggmux avimux matroskamux avsysvideosrc camerasrc fimcconvert'
```
Comment (kparichay) [inline packaging/nnstreamer.spec]: `videomixer` to support overlay of videos together?
Useful for displaying the results of object recognition.

Comment (kparichay) [reply packaging/nnstreamer.spec]: `videoflip` to adjust orientation of incoming input video for tizen/andriod applications.

File: packaging/nnstreamer.spec (hunk 1 above)
Comment (myungjoo) [reply packaging/nnstreamer.spec]: add compositor and videomixer, which are essential for writing demo-apps.

**Comment by jaeyun-jung** [comment]
V2 (2019-09-17)
- Added new element (v4l2src compositor videoflip videomixer audiomixer)

**Comment by myungjoo** [comment]
@again4you @wooksong @leemgs @jijoongmoon @helloahn @dongju-chae Anyone who has written nnstreamer pipelines/apps should review this!

**Comment by myungjoo** [comment]
Anyway, let's accept further reviews until this evening. We can add more elements anytime.

**Comment by myungjoo** [comment]
LGTM. (not marking it until this evening, though)

**Pull request merged**